app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag password_resets_path do
+        = hidden_field_tag :token, @token
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = label_tag :password, "New Password"
+          .row
+            .col-sm-4
+              = password_field_tag :password, nil, class: "form-control"
+        %fieldset.form-group.action
+          %input(type="submit" value="Reset Password" class="btn btn-default")
```

Comment:
If you are using Rails helpers in this form above, I would also use them here too. Make the code much readable.